packages/app-utils/src/components/ConnectionBootstrap.tsx

```diff
@@ -56,47 +59,63 @@ export function ConnectionBootstrap({
     [api, client]
   );
 
-  useEffect(
-    function listenForShutdown() {
-      if (connection == null) return;
+  useEffect(() => {
+    if (connection == null) return;
```

Comment:
We don't have a linter rule for this, but on longer effects we prefer to provide named functions. Gives a little more insight into what the effect lifecycle is handling without needing to read the implementation. This was previously named `listenForShutdown`.

Change back to a named function or update the name if there's a better fitting name